app/javascript/components/components/Header.js

```diff
@@ -28,6 +42,40 @@ class Header extends Component {
           )}
         </ul>
         </div>
+
+          <Navbar className='resposive-nav-bar' color="faded" light>
+            <NavbarBrand href="/" className="mr-auto">Home</NavbarBrand>
+            <NavbarToggler onClick={this.toggle} className="mr-2" />
+            <Collapse isOpen={this.state.isOpen} navbar>
+              <Nav navbar>
+                <NavItem>
+                  <NavLink href="/aboutus">About Us</NavLink>
```

Comment:
You will want your internal component navigation to use `<NavLink to="">`

Reply:
Sorry just saw this, however we are using both NavLink from router and from reactstrap. Reactstrap NavLink uses href and NavLink from router uses to. If this is confusing I can explain it better in person but we were running into issues with using both because they both used the same name so we had to add a tag to the one of them.